src/ValidationError.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Schemer;
+
+class ValidationError
+{
+    private /* const */ $messages = [
```

Comment:
Yeah, which I really don't like. Happy to change it if you can think of something better.